packages/odyssey-react-labs/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@okta/odyssey-typescript/tsconfig.react.json",
   "compilerOptions": {
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "jsxImportSource": "@emotion/react"
```

Comment:
We no longer need these lines 👍

Reply:
Removed these as well as the exports in `theme.ts` we're no longer using